x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_list_card.tsx

```diff
@@ -0,0 +1,183 @@
+                            <EuiBadge>{exceptionsList.created_by}</EuiBadge>{' '}
+                          </EuiFlexItem>
+                        </EuiFlexGroup>
+                      </EuiFlexItem>
+                      <EuiFlexItem style={{ flex: '1 1 auto' }}>
+                        <EuiFlexGroup gutterSize="xs" alignItems="center">
+                          <EuiFlexItem grow={false}>
+                            <EuiText>{'created by: '}</EuiText>
+                          </EuiFlexItem>
+                          <EuiFlexItem grow={false}>
+                            <EuiBadge>{exceptionsList.created_by}</EuiBadge>{' '}
+                          </EuiFlexItem>
+                        </EuiFlexGroup>
+                      </EuiFlexItem>
+                      <EuiFlexItem style={{ flex: '1 1 auto' }}>
+                        <EuiFlexGroup gutterSize="xs" alignItems="center">
+                          <EuiFlexItem grow={false}>
+                            <EuiText>{'created on: '}</EuiText>
+                          </EuiFlexItem>
+                          <EuiFlexItem grow={false}>
+                            <EuiBadge>{exceptionsList.created_at}</EuiBadge>
+                          </EuiFlexItem>
+                        </EuiFlexGroup>
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiPopover
+                  data-test-subj="exceptionsListCardOverflowActions"
+                  button={
+                    <EuiButtonIcon
+                      isDisabled={false}
+                      data-test-subj="exceptionsListCardOverflowActions"
+                      aria-label="Exception item actions menu"
+                      iconType="boxesHorizontal"
+                      onClick={onItemActionsClick}
+                    />
+                  }
+                  panelPaddingSize="none"
+                  isOpen={isPopoverOpen}
+                  closePopover={onClosePopover}
+                >
+                  <EuiContextMenuPanel
+                    size="s"
+                    items={[
+                      <EuiContextMenuItem
+                        key={'delete'}
+                        disabled={exceptionsList.list_id === 'endpoint_list' || readOnly}
+                        data-test-subj="exceptionsTableDeleteButton"
+                        icon={'trash'}
+                        onClick={() => {
+                          onClosePopover();
+                          handleDelete({
+                            id: exceptionsList.id,
+                            listId: exceptionsList.list_id,
+                            namespaceType: exceptionsList.namespace_type,
+                          })();
+                        }}
+                      >
+                        {'Delete exception list'}
```

Comment:
I know this is draft - but just in case! A couple strings in here need i18n translations.

Reply:
Fixed here: 8a4d040f87af63530ea74ed97d926cff11004922